core/domain/fs_domain_test.py

```diff
@@ -105,16 +107,108 @@ def test_independence_of_file_systems(self):
             fs2.get('abc.png')
 
 
-class DiskBackedFileSystemTests(test_utils.GenericTestBase):
+class DiskBackedFileSystemUnitTests(test_utils.GenericTestBase):
     """Tests for the disk-backed file system."""
 
+    def setUp(self):
+        super(DiskBackedFileSystemUnitTests, self).setUp()
+        self.user_id = 'abc@example.com'
+
     def test_get(self):
         fs = fs_domain.AbstractFileSystem(fs_domain.DiskBackedFileSystem(
             feconf.TESTS_DATA_DIR))
         self.assertTrue(fs.get('img.png'))
         with self.assertRaisesRegexp(IOError, 'No such file or directory'):
             fs.get('non_existent_file.png')
 
+    def test_commit(self):
+        fs = fs_domain.AbstractFileSystem(fs_domain.DiskBackedFileSystem(
```

Comment:
Maybe this step can be moved into setup? Every test seems to be doing this.

Reply:
Done.